projects/optic/package.json

```diff
@@ -112,5 +112,21 @@
     "update-notifier": "^5.1.0",
     "url-join": "^4.0.1",
     "whatwg-mimetype": "^3.0.0"
+  },
+  "pkg": {
+    "scripts": [
+      "build/commands/ruleset/upload.js"
+    ],
+    "assets": [
+      "../../node_modules/open/xdg-open",
+      "../../node_modules/vm2/lib/setup-sandbox.js"
```

Comment:
this file doesn't get a warning during the build, but if it's excluded you get a runtime error,

```
➜ dist/optic-macos-arm64 --help
pkg/prelude/bootstrap.js:1872
      throw error;
      ^

Error: File '/**/optic/node_modules/vm2/lib/setup-sandbox.js' was not included into executable at compilation stage. Please recompile adding it as asset or script.
```